Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt

```diff
@@ -98,8 +91,17 @@ class SubmissionTestResultPendingFragment : Fragment(R.layout.fragment_submissio
         super.onResume()
         binding.submissionTestResultContainer.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
         pendingViewModel.refreshDeviceUIState(refreshTestResult = !skipInitialTestResultRefresh)
-
         skipInitialTestResultRefresh = false
+        pendingViewModel.cwaWebExceptionLiveData.observe(this.viewLifecycleOwner) { exception ->
+            pendingViewModel.cwaWebExceptionLiveData.removeObservers(this.viewLifecycleOwner)
+            handleError(exception)
+        }
+    }
+
+    override fun onPause() {
+        super.onPause()
```

Comment:
Won't cause an issue, here, but FYI, it's considered best practice to call super.<incoming> (onCreate, onResume) before your own code, and super.<outgoing> (onPause, onDestroy) after your own code.

https://groups.google.com/g/android-developers/c/ndsrBsIchFc?pli=1

Reply:
switched the order